### PR TITLE
Fix deploy script for recent Bikeshed boilerplate updates

### PR DESCRIPTION
**Pull request by domenic**
- We don't use Title: in the source documents anymore, so scrape the H1: instead
- Work around https://github.com/tabatkins/bikeshed/issues/1162

### CODE REVIEW

**Comment by domenic** [comment]
tabatkins/bikeshed#1162 has been fixed but it seems like elsewhere in the header we prefer to omit quotes around attribute values so I'll leave that change in anyway.

File: resources.whatwg.org/build/deploy.sh [hunk 1]
```diff
@@ -76,7 +76,7 @@ echo ""
 # Living standard
 curl https://api.csswg.org/bikeshed/ -f -F file=@"$INPUT_FILE" \
      -F md-Text-Macro="SNAPSHOT-LINK $SNAPSHOT_LINK" \
-     > "$WEB_ROOT/index.html"
+     > "$WEB_ROOT/index.html";
```
Comment (annevk) [inline resources.whatwg.org/build/deploy.sh]: Nothing else here uses ";". Was this change intentional?
